app/screens/InfoScreen.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react"
+import {
+  ImageStyle,
+  TextStyle,
+  View,
+  ViewStyle,
+  FlatList,
+  ImageSourcePropType,
+  Image,
+} from "react-native"
+import { CarouselCard, Screen, Text } from "../components"
+import { useHeader } from "../hooks"
+import { TabScreenProps } from "../navigators/TabNavigator"
+import { colors, spacing } from "../theme"
+import { translate } from "../i18n"
+import { openLinkInBrowser } from "../utils/openLinkInBrowser"
+
+const IMAGE_WIDTH = 358
+
+const phoneNumber = "360-450-4752"
+
+const confImage = require("../../assets/images/info-conf.png")
+const irImage1 = require("../../assets/images/info-ir1.png")
+const irImage2 = require("../../assets/images/info-ir2.png")
+const irImage3 = require("../../assets/images/info-ir3.png")
+
+const carouselImages: ImageSourcePropType[] = [irImage1, irImage2, irImage3]
+
+export const InfoScreen: React.FunctionComponent<TabScreenProps<"Info">> = () => {
+  useHeader({ title: translate("infoScreen.title") })
+
+  // NOTE: this only works on a device, warning in sim
+  const callPhoneNumber = () => openLinkInBrowser(`tel:${phoneNumber}`)
+
+  return (
+    <Screen style={$root} preset="scroll" ScrollViewProps={{ showsVerticalScrollIndicator: false }}>
+      <View style={$content}>
+        <Text preset="screenHeading" tx="infoScreen.screenHeading" />
+      </View>
+
+      <FlatList
+        data={carouselImages}
+        keyExtractor={(_, index) => `info-image-${index}`}
+        bounces={false}
+        showsHorizontalScrollIndicator={false}
+        horizontal
+        pagingEnabled
+        decelerationRate="fast"
+        snapToInterval={IMAGE_WIDTH}
```

Comment:
@frankcalise curious if this will cause problems on different sized devices? I think we should take the device's screen into consideration here?